met-api/src/met_api/models/comment.py

```diff
@@ -146,3 +147,24 @@ def update(cls, submission_id, comment: CommentSchema, session=None) -> Comment:
         else:
             session.flush()
         return query.first()
+
+    @classmethod
+    def get_comments_by_survey_id(cls, survey_id):
+        """Get comments paginated."""
+        query = db.session.query(
+            Comment.id,
+            Comment.submission_date,
+            Comment.text,
+            User.first_name,
+            User.last_name,
+            Submission.reviewed_by
+        )\
+            .join(Submission, Submission.id == Comment.submission_id) \
```

Comment:
Just a reminder that this query can return more than 1 comment per submission (depending on the number of free text questions) this could be a problem or not, we need to discuss how to approach this with Marie Anik/Carly

Reply:
Good point, noted